src/index.ts

```diff
@@ -52,4 +94,22 @@ export = class implements RNS {
       return this._resolutionHelper.chainAddr(domain, chainId);
     }
   }
+
+  /**
+   * Checks if the given label subdomain is available under the given domain tree
+   * 
+   * @throws
+   * Throws an error if the given domain is not .rsk, if it is a subdomain or if it is not alphanumeric
+   * 
+   * @param domain - Parent .rsk domain. ie: wallet.rsk
+   * @param label - Subdomain to check if is available. ie: alice
+   * 
+   * @returns
+   * true if available, false if not
+   */
+  async checkSubdomainAvailability(domain: string, label: string): Promise<boolean> {
```

Comment:
I prefer `isSubdomainAvailable`, what do you think?

Reply:
Like it